oaknational/Oak-Web-Application
# fix: update to mv_curriculum_units_5

## Description
- Update the API endpoint for the curriculum visualiser to `mv_curriculum_units_5`
- Fixes missing tier units in Secondary Science
- Fixes missing tier units in Secondary Maths
- Fixes missing units in KS4 History and Music

## How to test

1. Go to https://deploy-preview-1986--oak-web-application.netlify.thenational.academy/teachers/curriculum/music-secondary-aqa/units
2. You should see units for all years
3. Go to https://deploy-preview-1986--oak-web-application.netlify.thenational.academy/teachers/curriculum/history-secondary-edexcel/units
4. You should see units for all years
6. Go to https://deploy-preview-1986--oak-web-application.netlify.thenational.academy/teachers/curriculum/science-secondary-ocr/units
7. Scroll down to year 10
8. You should see unit 5: "States of matter"
9. Click on the tier "Higher"
10. You should still see unit 5: "States of matter"

NOTE: Could not recreate some issues for old MV, possible due to data changes on production. Screenshots added below for issues that could be reproduced.

## Screenshots

How it used to look (delete if n/a):
<img width="1263" alt="Screenshot 2023-10-08 at 14 32 41" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/fd761bf7-2c3b-4c98-970c-206fbb43547b">

How it should now look:
<img width="1263" alt="Screenshot 2023-10-08 at 14 33 30" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/f27d0a75-c9e0-4b8a-9ab3-6b48155c3c4d">


## Checklist
